html/cross-origin-opener-policy/resources/postback.html

```diff
@@ -5,6 +5,5 @@
 const bc = new BroadcastChannel(channelName);
 window.addEventListener("message", event => {
   bc.postMessage(event.data);
-  window.parent.close();
```

Comment:
I think it's worth explaining why close is not called here.

Reply:
Thanks Anne! I'm changing this to: the main test sending a message to the popup to trigger the closure during the cleanup. I'll push that new version to give you an idea of what I'm meaning here. I still need to update the other cases (as pointed out by your other comments).

I'm not entirely sure if the iframe is able to close its parent/top in all cases though, i.e. interpreting https://html.spec.whatwg.org/multipage/window-object.html#dom-window-close correctly. I think it is, but if not, I would add an extra hop where postback sends a message to the parent to trigger the closure...

Reply:
There's a couple cases where neither gets to close it from script. This is mostly best effort for those running the tests manually.